### PR TITLE
Build against FAWE 2.0.0 and reuse WE's log manager

### PR DESCRIPTION
**Pull request by NotMyFault**
## Description
Additionally I dropped our Log4J implementation and switched to the LogManagerCompat WorldEdit provides.

## Submitter Checklist

- [X] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [X] Ensure that the pull request title represents the desired changelog entry.
- [X] I tested my changes and approved their functionality.
- [X] I ensured my changes do not break other parts of the code
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md)

## Maintainer checklist

Before the changes are marked as `Ready for merge`: 

- [x] There is at least 1 approval for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [X] Changelog entries in the PR title are correct
- [X] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional modification steps from users, e.g. for permission changes, make sure the entry is added to the pull request description so it can be appended to the changelog.

### CODE REVIEW

**Review by Aurelien30000 — APPROVED** [review]
Easier than last time 😅

**Comment by NotMyFault** [comment]
Yup, based on the little testing I did, everything works. Luckily you already addressed world heights before.

Though I'm tempted to wait until we release 2.0.0 before merging this PR, so renovate can pick it up.

**Pull request merged**
